pydis_site/templates/events/pages/code-jams/the-code-style-guide.html

```diff
@@ -0,0 +1,299 @@
+        But if you still do, you should avoid characters that may look like zero or one in some fonts:
+        "O" (uppercase o), "l" (lowercase L), and "I" (uppercase i).
+    </p>
+
+    <h3>Operators</h3>
+    <p>
+        If you have a chain of mathematic operations that you split into multiple lines,
+        you should put the operator at the beginning of the line and not to the end of the line.
+    </p>
+    <pre>
+        <code class="language-python">
+            # No
+            result = (
+                1 +
+                2 *
+                3
+            )
+
+            # Yes
+            result = (
+                1
+                + 2
+                * 3
+            )
+        </code>
+    </pre>
+    <p>If you compare against <code>None</code>, you should use <code>is</code> and <code>is not</code>, but not compare equality.</p>
+    <pre>
+        <code class="language-python">
+            # No
+            if variable == None:
+                print("Variable is None")
+
+            # Yes
+            if variable is None:
+                print("Variable is None")
+        </code>
+    </pre>
+    <p>
+        You should prefer using <code><item one> is not <item two></code> over <code>not <item one> is <item two></code>.
+        Using second makes it harder to understand the expression.
+    </p>
+    <pre>
+        <code class="language-python">
+            # No
+            if not variable is None:
+                print("Variable is not None")
+
+            # Yes - it is much easier to read and understand this than previous
+            if variable is not None:
+                print("Variable is not None")
+        </code>
+    </pre>
+
+    <h3>Imports</h3>
+    <p>Imports should be at top of the file, the only things that should be before them are module comments and docstrings.</p>
+    <p>You shouldn't import multiple modules in one line, but give each module import its own line instead.</p>
+    <pre>
+        <code class="language-python">
+            # No
+            import pathlib, os
+
+            # Yes
+            import os
+            import pathlib
+        </code>
+    </pre>
+    <p>Wildcard imports should be avoided in most cases. They make unclear where what comes from.</p>
+    <pre>
+        <code class="language-python">
+            # No
+            from pathlib import *
+
+            # Yes
+            from pathlib import Path
+        </code>
+    </pre>
+    <p>You should use <b><a href="https://pycqa.github.io/isort/">isort</a><b> imports order specification, what means:</p>
+    <ul>
+        <li>
+            <b>Group by type:</b> order of import types should be <code>__future__</code> imports, standard library imports,
+            third-party library imports, and finally project imports.
+        </li>
+        <li>
+            <b>Group by import way:</b> inside each group, first should come imports in format <code>import &lt;package&gt;</code>
```

Comment:
```suggestion
            <b>Group by import method:</b> inside each group, first should come imports in format <code>import &lt;package&gt;</code>
```
I don't think way fits here; should also be changed below if you change this